### PR TITLE
update `eventSystem/streams`

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- remove cuda types and header
- use alpaka methods instead of cuda

Merge after:
- [x] issue #4

### CODE REVIEW

File: src/libPMacc/include/eventSystem/streams/EventStream.hpp [hunk 1]
```diff
@@ -54,17 +56,21 @@ class EventStream
...
     }
 
     /**
      * Returns the cudaStream_t object associated with this EventStream.
      * @return the internal cuda stream object
      */
-    cudaStream_t getCudaStream() const
+    const alpaka::AccStream& getCudaStream() const
+    {
+        return *stream.get();
```
Comment (erikzenker) [inline src/libPMacc/include/eventSystem/streams/EventStream.hpp]: '*' to dereference is enough. Also for the rest of the file.

File: src/libPMacc/include/eventSystem/streams/EventStream.hpp [hunk 2]
```diff
@@ -42,9 +40,13 @@ class EventStream
      * Constructor.
      * Creates the cudaStream_t object.
      */
-    EventStream() : stream(NULL)
+    EventStream() :
+        stream(
+            alpaka::AccStream(
+                new Environment<DIM1>::get().DeviceManager().getAccDevice()
```
Comment (erikzenker) [inline src/libPMacc/include/eventSystem/streams/EventStream.hpp]: Is this 'new' on the correct position ?

Comment (psychocoderHPC) [inline src/libPMacc/include/eventSystem/streams/EventStream.hpp]: well spotted: it must moved one line up

**Comment by psychocoderHPC** [comment]
rebased, I add all fixes and changes in a new pull request

**Comment by erikzenker** [comment]
I gonna merge them all

**Pull request merged**